Breadcrumbs.py

```diff
@@ -108,4 +129,186 @@ def get_points_by_row(row):
               breadcrumbs[index_of_breadcrumb] = breadcrumbs[index_of_breadcrumb][0:length_of_trim]
           break
       number_of_characters_left -= current_length
-    view.set_status('breadcrumbs',separator.join(breadcrumbs+['']))
+  return breadcrumbs
+
+
+def copy(view, text):
+  sublime.set_clipboard(text)
+  view.hide_popup()
+  sublime.status_message('Breadcrumbs copied to clipboard')
+
+
+class BreadcrumbsCommand(sublime_plugin.EventListener):
+
+  def on_selection_modified(self, view):
+    view.erase_status('breadcrumbs')
+
+    settings = sublime.load_settings('Breadcrumbs.sublime-settings')
+    default_statusbar_enabled = settings.get('show_breadcrumbs_in_statusbar', True)
+    statusbar_enabled = view.settings().get('show_breadcrumbs_in_statusbar', default_statusbar_enabled)
+
+    if statusbar_enabled:
+      current_row = view.rowcol(view.sel()[0].b)[0]
+      breadcrumbs = make_breadcrumbs(view, current_row, shorten=True)
+
+      if breadcrumbs is not None and len(breadcrumbs) > 0:
+        view.set_status('breadcrumbs', get_separator(view).join(breadcrumbs))
+
+
+class BreadcrumbsPopupCommand(sublime_plugin.TextCommand):
+
+  def run(self, edit):
+    stylesheet = '''
+      p {
+        margin-top: 0;
+      }
+      a {
+        font-family: system;
```

Comment:
The style here is copied from `show_scope_name.py`. The `system` keyword is undocumented, but seems to tell it to use the same font as the UI. Which makes a lot of sense, it would be weird for to have lists of fallback fonts in hope that one of them matches the font used throughout the application. Not sure why they didn't use a css var here though, but it works.

> Anyway it would probably be a good idea to have some fallback font names listed as a fallback

So, no need for this.

> I've tried to remove this line and it still works without it.

It does, but it would inherit the editor/content font, not the UI font. Which is acceptable, but I wanted to match the pre-existing UI for show_scope_name, which has very similar usage and content.